runtime/strategies/coalesce-recipes.js

```diff
@@ -119,14 +120,13 @@ export class CoalesceRecipes extends Strategy {
       }
 
       onHandle(recipe, handle) {
-        if ((handle.fate !== 'use' && handle.fate !== '?')
+        if (!coalescableFates.includes(handle.fate)
```

Comment:
So this will coalesce use + use, use + create, create + use, create + create, right?

What's the resulting handle fate in each case? Could you add that as a comment here?

Reply:
good point - if the handles were `use` + `use`, I should keep the `use` fate. updated and added comment.